docs/platform/enterprise-flex/data-plane.md

```diff
@@ -0,0 +1,452 @@
+| ---------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`           | Required  | The name of your region in Airbyte. We reccomend as best practice that you include the cloud provider  (if applicable), and actual region in the name. |
+| `organizationId` | Required  | Your Airbyte organization ID. To find this in the UI, navigate to `Settings` > `General`.                                             |
+| `enabled`        | Optional  | Defaults to true. Set this to `false` if you don't want this region enabled.                                                             |
+
+For additional request examples, see [the API reference](https://reference.airbyte.com/reference/regions#/).
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+Make note of your `regionId`. You need it to create a data plane.
+
+```json title="200 Successful operation"
+{
+  //highlight-next-line
+  "regionId": "uuid-string",
+  "name": "region-name",
+  "organizationId": "org-uuid-string",
+  "enabled": true,
+  "createdAt": "timestamp-string",
+  "updatedAt": "timestamp-string"
+}
+```
+</details>
+
+## 2. Create a data plane {#step-2}
+
+Once you have a region, you create a data plane within it.
+
+<details>
+  <summary>Request</summary>
+
+Send a POST request to /v1/dataplanes.
+
+```bash
+curl -X POST https://api.airbyte.com/v1/dataplanes \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/json" \
+  -d '{
+    "name": "aws-us-east-1",
+    "regionId": "00000000-0000-0000-0000-000000000000"
+  }'
+```
+
+Include the following parameters in your request.
+
+| Body parameter | Required? | Description                                                                                                         |
+| -------------- | --------- | ------------------------------------------------------------------------------------------------------------------- |
+| `name`         | Required  | The name of your data plane. For simplicity, you might want to name it based on the region in which you created it. |
+| `regionId`     | Optional  | The region this data plane belongs to.                                                                              |
+
+For additional request examples, see [the API reference](https://reference.airbyte.com/reference/dataplanes#/).
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+Make note of your `dataplaneId`, `clientId` and `clientSecret`. You need these values later to deploy your data plane on Kubernetes.
+
+```json title="200 Successful operation"
+json
+{
+  "dataplaneId": "uuid-string",
+  "clientId": "client-id-string",
+  "clientSecret": "client-secret-string"
+}
+```
+
+</details>
+
+## 3. Associate a region to a workspace {#step-3}
```

Comment:
I think this should be the last step, since this will break your syncs if this is a workspace with existing connections in it